src/main/scala/no/ndla/audioapi/model/api/AudioMetaInformation.scala

```diff
@@ -24,6 +24,7 @@ case class AudioMetaInformation(
     @(ApiModelProperty @field)(description = "Tags for this audio file") tags: Tag,
     @(ApiModelProperty @field)(description = "The languages available for this audio") supportedLanguages: Seq[String],
     @(ApiModelProperty @field)(description = "Type of audio. 'standard', or 'podcast'.") audioType: String,
-    @(ApiModelProperty @field)(description = "Meta information about podcast, only applicable if audioType is 'podcast'.") podcastMeta: Option[PodcastMeta]
+    @(ApiModelProperty @field)(description = "Meta information about podcast, only applicable if audioType is 'podcast'.") podcastMeta: Option[PodcastMeta],
+    @(ApiModelProperty @field)(description = "Manuscript for the podcast") manuscript: Option[Manuscript]
```

Comment:
Burde vi returnerer serien en podcast er en del av, eller ser du for deg at alt av serie-redigering skjer på seriesida?

Reply:
Kan legge til :+1: